region/brinstar/red/Red Brinstar Fireflea Room.json

```diff
@@ -573,6 +535,36 @@
       ],
       "note": "Use the Fireflea when it moves to the right to cross the first gap."
     },
+    {
+      "link": [2, 3],
+      "name": "HiJump and Wall Jump",
+      "requires": [
+        "HiJump",
+        "canPreciseWalljump",
+        "canIframeSpikeJump",
+        {"spikeHits": 2}
+      ],
+      "note": "Jump from on top of the lower spikes, wall jump on the overhang, and land on the upper spikes."
+    },
+    {
+      "link": [2, 3],
+      "name": "HiJump Damage Boost",
+      "requires": [
+        "HiJump",
+        "canTrickyJump",
+        {"enemyDamage": {
+          "enemy": "Fireflea",
+          "type": "contact",
+          "hits": 1
+        }},
+        "canHorizontalDamageBoost",
+        {"or": [
+          "canIframeSpikeJump",
+          {"spikeHits": 1}
+        ]}
+      ],
+      "note": "Damage boost off of the Fireflea in order to get onto the upper spikes."
```

Comment:
This jump felt a little easier than the hijumpless version, but still harder than the fast Waver boost.
I'd rather wait for the waver to come back than set this up.  But I don't the timing for when the waver comes back to a good spot yet.